frontend/src/app/components/complete-sprint/complete-sprint-dialog/complete-sprint-dialog.component.ts

```diff
@@ -31,17 +33,24 @@ export class CompleteSprintDialogComponent implements OnInit {
   }
 
   onSubmit(): void {
-    this.sprintService.completeSprint(this.sprint.id).subscribe();
+    this.sprintService.completeSprint(this.sprint.id).subscribe(() => {
+      this.changeOutside();
+    });
     this.dialogRef.close();
   }
 
+  changeOutside(): void {
+    this.backlogService.changeBacklog();
+    this.sprintService.changeDeleteSprint(this.sprint.id);
+  }
+
   openIssue(): number {
-    return this.sprint.tasks.filter((t) => t.stateId == 1 || t.stateId == 3)
+    return this.sprint.tasks.filter((t) => t.stateId == 1 || t.stateId == 2)
```

Comment:
you should have an enum on the client side that will represent these numbers, for now it's not clear what each number means